CHANGES.rst

```diff
@@ -4,6 +4,8 @@ Changelog
 4.2 (unreleased)
 ----------------
 
+- Replace urllib.quote with six.moves.urllib.parse.quote
```

Comment:
Does this fix the ZMI for this package? (If yes, this should be mentioned in the change log.)